cmd/explaintest/t/integration_partition.test

```diff
@@ -0,0 +1,14 @@
+# TestIssue54746
```

Comment:
Remove integration test in release-6.5

Reply:
Why? It was in the tests/integrationtests, but I moved it to the explaintest to keep it covered also in release-6.5.
Or do you want me to convert it to a unit test (..._test.go)?